### PR TITLE
UAVCAN: Optical Flow: Bug fix missing integration_timespan field

### PR DESCRIPTION
**Pull request by JacobCrabill**
…optical_flow topic).

This is pretty straightforward.  The uORB topic `optical_flow` includes a field `integration_timespan`, which is used within EKF2 (after being copied to the `dt` field of a `flowSample` object) for a number of calculations, including for the variable `flow_innov`.  The UAVCAN sensor driver for optical flow devices (HereFlow) did not copy this field to the topic before publishing (but it does exist in the CAN message frame).  This resulted in `flow_innov` always being 0.  The fix: copy the field (ensuring consistent units).

### CODE REVIEW

**Comment by TSC21** [comment]
@JacobCrabill please run `make format` to fix the style and amend the commit.

File: src/drivers/uavcan/sensors/flow.cpp [hunk 1]
```diff
@@ -68,6 +68,8 @@ UavcanFlowBridge::flow_sub_cb(const uavcan::ReceivedDataStructure<com::hex::equi
 	optical_flow_s flow{};
 
 	flow.timestamp = hrt_absolute_time();
+	flow.integration_timespan = 1.e6f*msg.integration_interval; // s -> micros
```
Comment (bresch) [inline src/drivers/uavcan/sensors/flow.cpp]:
```suggestion
	flow.integration_timespan = 1.e6f * msg.integration_interval; // s -> micros
```

File: src/drivers/uavcan/sensors/flow.cpp [hunk 2]
```diff
@@ -68,6 +68,8 @@ UavcanFlowBridge::flow_sub_cb(const uavcan::ReceivedDataStructure<com::hex::equi
...
+
```
Comment (bresch) [inline src/drivers/uavcan/sensors/flow.cpp]:
```suggestion
```

**Comment by JacobCrabill** [comment]
Fixed the formatting issues
Should be all set

File: src/drivers/uavcan/sensors/flow.cpp (hunk 1 above)
Comment (JacobCrabill) [reply src/drivers/uavcan/sensors/flow.cpp]: Fixed

File: src/drivers/uavcan/sensors/flow.cpp (hunk 2 above)
Comment (JacobCrabill) [reply src/drivers/uavcan/sensors/flow.cpp]: Fixed

**Comment by MaEtUgR** [comment]
@JacobCrabill Just a hint: if you set the git identity on your machine to the mail address you use on GitHub it will assign it to your profile and show the profile picture next to the commit and you'll appear in the GitHub statistics.
e.g. `$ git config --global user.email example@example.com`

**Comment by JacobCrabill** [comment]
@MaEtUgR thanks for the tip.  We changed domain names at Volans-i a while back and I forgot to update my email address in my account.

**Comment by JacobCrabill** [comment]
@dagar , @bresch should this be rebased on / merged with the current "beta" branch (v1.10-beta4) instead of master in order to include in the 1.10 release?  Or, is the current head of master going to end up in v1.10?

**Comment by julianoes** [comment]
@JacobCrabill I think we can cherry-pick it into the release branch.

**Pull request merged**